control-plane-operator/controllers/hostedcontrolplane/v2/cvo/ibm/preparepayload.go

```diff
@@ -0,0 +1,126 @@
+package ibm
```

Comment:
why is this called ibm?

Reply:
this is only called for ibm/powervs platform
https://github.com/openshift/hypershift/pull/5125/files#diff-213db240d2bbec8b38a0bc813f1683a1d16a6790e3e905038c574db9a9e5321dR26-R30